alembic/versions/5a1fada2e223_initial_schema.py

```diff
@@ -0,0 +1,95 @@
+"""Initial schema
+
+Revision ID: 5a1fada2e223
+Revises: dbba5d52c8d9
+Create Date: 2025-06-15 11:35:51.379087
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "5a1fada2e223"
+down_revision: Union[str, None] = "dbba5d52c8d9"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    """Upgrade schema."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "users",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("login", sa.String(length=255), nullable=False),
+        sa.Column("name", sa.String(length=255), nullable=False),
+        sa.Column("email", sa.String(length=255), nullable=False),
+        sa.PrimaryKeyConstraint("id"),
+        sa.UniqueConstraint("login"),
+    )
+    op.create_table(
+        "versions",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("file_id", sa.String(length=255), nullable=False),
+        sa.Column("s3_id", sa.String(length=255), nullable=False),
+        sa.Column("version_number", sa.Integer(), nullable=False),
+        sa.Column("stage", sa.String(length=50), nullable=False),
+        sa.Column("change_description", sa.String(length=1000), nullable=False),
+        sa.Column("repository_id", sa.Integer(), nullable=False),
+        sa.Column("created_by", sa.Integer(), nullable=False),
+        sa.Column("created_at", sa.DateTime(), nullable=False),
+        sa.Column("checksum", sa.String(length=64), nullable=False),
+        sa.ForeignKeyConstraint(["created_by"], ["users.id"], ondelete="CASCADE"),
+        sa.ForeignKeyConstraint(
+            ["repository_id"], ["repositories.id"], ondelete="CASCADE"
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_table(
+        "envs",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("name", sa.String(length=255), nullable=False),
```

Comment:
i'm just being nit picky here, but i think naming the column as `key` makes more sense in this context than `name`(?)